internal/options/options.go

```diff
@@ -74,8 +79,9 @@ func (o *Options) AddFlags() {
 	o.flags.IntVar(&o.ExporterPort, "exporter-port", 8081, "Port to expose kube-events-exporter own metrics on.")
 	o.flags.BoolVar(&o.Version, "version", false, "kube-events-exporter version information")
 
-	o.flags.StringArrayVar(&o.EventTypes, "event-types", []string{EventTypesAll}, "Event types filter. Defaults to all types.")
-	o.flags.StringArrayVar(&o.InvolvedObjectNamespaces, "involved-object-namespaces", []string{metav1.NamespaceAll}, "Events involved object namespace filter. Defaults to all namespaces.")
+	o.flags.StringArrayVar(&o.EventTypes, "event-types", []string{EventTypesAll}, "List of included Event types. Defaults to all types.")
```

Comment:
Included sounds strange wording to me, I more often see "Allowed" or "Filtered".

Reply:
According to our discussion, I went with "Allowed".